front-end/e-learning/src/components/App.js

```diff
@@ -1,16 +1,23 @@
-import React from 'react';
+/** @format */
 
-import Login from './Login';
+import React, { useState } from "react";
+import { BrowserRouter as Router, Route, Routes } from "react-router-dom";
 
-class App extends React.Component {
+import Login from "./Login";
+import Dashboard from "./Dashboard";
 
-  render() {
-    return (
-      <div>
-        <Login />
-      </div>
-    );
-  }
-}
+const App = () => {
+  // const [token, setToken] = useState('')
```

Comment:
remove if not needed

Reply:
I'm not yet sure how and where to save the token, so I commented that out, it might be used in the future.
But I can remove it ma'am if necessary. Thank you.